### PR TITLE
IS task MMD model conversion to mlflow

### PR DESCRIPTION
**Pull request by gauravrajguru**
Pipeline with online deployment test:
https://ml.azure.com/experiments/id/89f43ebe-93b3-48a9-b138-e2d014a91429/runs/ce7abb83-67e3-4a23-8869-81b1566ab209?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/grajguru-resource-group/providers/Microsoft.MachineLearningServices/workspaces/grajguru-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Batch deployment job:
https://ml.azure.com/experiments/id/0eace021-d7d2-4ea4-8491-c66ca81ff456/runs/batchjob-8c8eb9dc-a5b8-4ec4-939b-b8ecef92629f?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/grajguru-resource-group/providers/Microsoft.MachineLearningServices/workspaces/grajguru-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

### CODE REVIEW

**Review by rjaincc — APPROVED** [review]

File: assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/detection_predict.py [hunk 1]
```diff
@@ -101,8 +106,29 @@ def _is_valid_url(text: str) -> bool:
...
+            x = float(x) / image_size[0]
+            y = float(y) / image_size[1]
+            normalized_points.extend([x, y])
+        normalized_polygon.append(np.array(normalized_points))
+    return normalized_polygon
+
+
 class ImagesDetectionMLflowModelWrapper(mlflow.pyfunc.PythonModel):
-    """Convert AutoML Images models to MLflow."""
+    """MLFlow model wrapper for Images Detection models."""
```
Comment (novaturient95) [inline assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/detection_predict.py:131]:
```suggestion
    """MLflow model wrapper for Images Detection models."""
```

**Review by novaturient95 — DISMISSED** [review]

**Review by MadhuM02 — DISMISSED** [review]
lgtm. Make sure to test the deployments.

File: assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/detection_predict.py (hunk 1 above)
Comment (gauravrajguru) [reply assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/detection_predict.py]: Thanks. Addressed it.

**Comment by gauravrajguru** [comment]
> lgtm. Make sure to test the deployments.

Added test runs in descriptions.

**Review by rjaincc — APPROVED** [review]

**Pull request merged**